main.py

```diff
@@ -78,12 +78,19 @@
                         help='The name of the requested chain',
                         choices=chain_dict.keys(),
                         )
+    parser.add_argument('--type',
+                        choices=("excel", "csv"),
+                        default='excel',
+                        
+        help="a switch flag to set set the output file to a CSV file",
```

Comment:
We can make it more informative, e.g.:
'The extension/type of the promotions output file'.
You wrote 'set' twice.